resources/views/print/print/print.blade.php

```diff
@@ -41,6 +41,7 @@
                     <p>
                         <label>
                             <input type="checkbox" name="use_free_pages" id="use_free_pages"
+                                @if(Auth::user()->freePages->isEmpty()) disabled @endif
```

Comment:
We could also hide this field here. Would be more cleaner.

Reply:
I made it.
Only thing is if the user has only expired free pages, the checkbox is still shown - or not?

Reply:
I think it should not. So we will need a new member function, something like hasActivePages (or some better name) and call it there.

Reply:
Done.